README.md

```diff
@@ -129,6 +87,9 @@ To decode and verify it, do:
 $ echo eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo= | python -m base64 -d
```

Comment:
maybe mention how to decode using system base64 also, since it is used to encode?

> 
```shell script
$ echo eyJzbGFjay10b2tlbiI6ICIxMjMtbXktc2VjcmV0LWFwaS1rZXkifQo= | base64 --decode
{"slack-token": "123-my-secret-api-key"}
```

Reply:
Agree - added your as default, then changed the python to being the "alternative"